From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

siriusk-MSFT commented:
I am Microsoft employee (alias siriusk), so CLA is not required.

deneha commented:
@siriusk-MSFT Is a MSFT employee.  No cla-required.

WilliamAntonRohm committed:
Merge pull request #31 from MicrosoftDocs/master

come on in

WilliamAntonRohm committed:
Merge pull request #36 from MicrosoftDocs/master

bring it on

rachel-msft committed:
server parameters for postgres, new concepts, new portal

rachel-msft committed:
Update howto-configure-server-parameters-using-portal.md

rachel-msft committed:
Update concepts-server-parameters.md

ktoliver committed:
edit pass /includes/storage-sync-files-change-detection.md

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-change-detection

ktoliver committed:
Update storage-sync-files-change-detection.md

line spacing

ktoliver committed:
Update storage-sync-files-change-detection.md

fix line spacing

jpallavi committed:
Update backup-azure-configure-reports.md

update with new steps based on diagnostic settings

jpallavi committed:
Add files via upload

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-configure-reports.md

jpallavi committed:
Delete enable-diagnostics.png

jpallavi committed:
Add files via upload

jpallavi committed:
Delete enable-diagnostics.png

jpallavi committed:
Add files via upload

ktoliver committed:
Update storage-sync-files-change-detection.md

more edits

ktoliver committed:
Update storage-sync-files-change-detection.md

rachel-msft committed:
Update TOC.yml

rachel-msft committed:
Update concepts-server-parameters.md

rachel-msft committed:
Update howto-configure-server-parameters-using-portal.md

rachel-msft committed:
Update howto-configure-server-parameters-using-portal.md

ktoliver committed:
Update storage-sync-files-change-detection.md

Server Endpoint > server endpoint; Azure Files share > Azure file share

ktoliver committed:
Update storage-sync-files-change-detection.md

Change sentence to exploring USN.

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

ccompy committed:
Merge pull request #8 from MicrosoftDocs/master

update my master

ccompy committed:
embedded a video link

WilliamAntonRohm committed:
Merge pull request #40 from MicrosoftDocs/master

meow

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

bwren committed:
log analytics changes article

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
draft/reorg of json indexing

HeidiSteen committed:
more edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HJToland3 committed:
image update

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

bwren committed:
Log Analytics changes

bwren committed:
toc

bwren committed:
link fix

WilliamAntonRohm committed:
Merge pull request #41 from MicrosoftDocs/master

howdy newbies!

WilliamAntonRohm committed:
Merge pull request #42 from MicrosoftDocs/master

more incoming

Commit:
Removed wad section

WAD section was added in the early releases of POA to facilitate support of SF versions 5.6*, 5.5*.
Since these versions are no more officially supported by SF, cleaning up the documentation.

bwren committed:
fixes

CarlRabeler committed:
danimir

Commit:
Updated Azure storage services comparison

CarlRabeler committed:
date

carolinacmoravia committed:
C6869: Hardcoded EN locale in link 

Hello, @mrbullwinkle,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

Commit:
freshness

carolinacmoravia committed:
C6883: Hardcoded EN locale in link

Hello, @mrbullwinkle,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

Commit:
borders on images

carolinacmoravia committed:
C6890: Hardcoded EN locale in link 

Hello, @MSSedusch,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

rachel-msft committed:
add server parameters info to concepts servers, new portal doc

billmath committed:
adding forgerock

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

billmath committed:
updating

rachel-msft committed:
merged date diff on servers concepts

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into newpgserverparam

HeidiSteen committed:
Json blob indexer doc updates

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

WilliamAntonRohm committed:
Merge pull request #43 from MicrosoftDocs/master

come on in

bwren committed:
typo

HeidiSteen committed:
Merge branch 'heidist-master' of https://github.com/HeidiSteen/azure-docs-pr; branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edit

Commit:
fix bug in filter parameters

curtand committed:
remove unneeded classic portal article

rachel-msft committed:
Update howto-configure-server-parameters-using-portal.md

bwren committed:
fices

Commit:
adding redirect

raiye committed:
Update cloud-services-guestos-msrc-releases.md

rjagiewich committed:
Merge pull request #27265 from curtand/domain1107

remove unneeded classic portal article

rjagiewich committed:
Merge pull request #27218 from Blackmist/freshness

freshness

WilliamAntonRohm committed:
undoing mistaken author name change (from PR 25886)

PRMerger19 committed:
Merge pull request #27264 from cbrooksmsft/patch-4

fix bug in filter parameters

rachel-msft committed:
Update concepts-servers.md

WilliamAntonRohm committed:
undoing mistaken author name changes (from PR 24678)

SnehaGunda committed:
Updating cmdlet

Commit:
Merge pull request #27274 from raiye/patch-36

Update cloud-services-guestos-msrc-releases.md

rmca14 committed:
Merge pull request #27091 from HeidiSteen/heidist-master

AzSearch JSON blob parsing

Commit:
Update table

PRMerger14 committed:
Merge pull request #27281 from SnehaGunda/policy

Updating cmdlet

ccompy committed:
updated url

Commit:
updating stale topics, some recently edited, eliminating refs to classic portal

tfosmark committed:
Merge pull request #27122 from bwren/la-changes

Log Analytics changes in upgraded workspace

PRMerger16 committed:
Merge pull request #27285 from DeniseMak/11-8-language

Update table

rmca14 committed:
Merge pull request #27269 from georgewallace/revertjava

storage : adding redirect for deleted file

PRMerger10 committed:
Merge pull request #26837 from ccompy/insert-video

Insert video

PRMerger13 committed:
Merge pull request #27276 from WilliamAntonRohm/W-hdi-fix-apache-PM-25886

undoing mistaken author name change (from PR 25886)

Commit:
Merge pull request #27282 from WilliamAntonRohm/W-hdi-fix-PM-24678

undoing mistaken author name changes (from PR 24678)

Commit:
add checkbox for es-ES

mwinkle committed:
Update release-notes-sprint-1.md

Adding links to sample projects

kydeeds committed:
Updating Azure Gov Pairing

PRMerger12 committed:
Merge pull request #27295 from DeniseMak/11-6-checkbox-sp

add checkbox for es-ES

curtand committed:
removing duplicated classic portal content

curtand committed:
removed from TOC

gbowerman committed:
Update the instance count change section

Commit:
Merge pull request #27296 from kydeeds/patch-3

Updating Azure Gov Pairing

Commit:
titles and intro

CarlRabeler committed:
Update sql-database-automatic-tuning.md

Commit:
Merge pull request #27300 from gbowerman/patch-19

Update the instance count change section

mwinkle committed:
Update release-notes-sprint-1.md

mimig1 committed:
Cosmos DB: Additional rewording from Rajeev

Commit:
acrolinx

Commit:
Merge pull request #27221 from carolinacmoravia/patch-447

C6883: Hardcoded EN locale in link

Commit:
Merge pull request #27212 from carolinacmoravia/patch-446

C6869: Hardcoded EN locale in link

Commit:
acrolinx

PRMerger12 committed:
Merge pull request #27294 from mwinkle/patch-4

Update release-notes-sprint-1.md

PRMerger14 committed:
Merge pull request #27302 from mimig1/patch-158

Cosmos DB: Additional rewording from Rajeev

GitHubber17 committed:
Merge pull request #27251 from rachel-msft/newpgserverparam

Newpgserverparam

CarlRabeler committed:
Update sql-database-automatic-tuning.md

GitHubber17 committed:
Merge pull request #27289 from bradsev/update-stale-topics-nov2017

updating stale topics, some recently edited, elim refs to classic portal

GitHubber17 committed:
Merge pull request #27298 from curtand/office1108

removing duplicated classic portal content

GitHubber17 committed:
Merge pull request #27301 from cynthn/titles

Titles and intro

CarlRabeler committed:
Update sql-database-automatic-tuning.md

GitHubber17 committed:
Merge pull request #27204 from CarlRabeler/tuning

danimir

Commit:
Corrected the link for localization

removed en-us from link

jpallavi committed:
Update backup-azure-configure-reports.md

jpallavi committed:
Update backup-azure-configure-reports.md

Commit:
Update backup-azure-vms-troubleshoot.md

Added private preview information to troubleshoot

Commit:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
Update backup-azure-arm-vms-prepare.md

Commit:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
Update backup-azure-vms-troubleshoot.md

dominicbetts committed:
Update architecture diagram

Commit:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
Update backup-azure-vms-troubleshoot.md

Commit:
Update backup-azure-arm-vms-prepare.md

Commit:
Merge pull request #27229 from carolinacmoravia/patch-448

C6890: Hardcoded EN locale in link

Commit:
updating daemon config suggesstions

Commit:
spelling fix

jpallavi committed:
Add files via upload

jpallavi committed:
Update backup-azure-configure-reports.md

Commit:
updated formatting

Commit:
numbering fix

jpallavi committed:
Update backup-azure-configure-reports.md

nsoneji committed:
minor text scrub to highlight the download link

Commit:
updated quoted string to regular string

Commit:
Merge pull request #27314 from sayantancs/updateipv6

updating daemon config suggesstions

PRMerger10 committed:
Merge pull request #27315 from nsoneji/master

minor text scrub to highlight the download link

Commit:
Merge pull request #27240 from billmath/idp

adding forgerock

PatrickBue committed:
changed example images to fit on page

Commit:
Merge pull request #27312 from trinadhk/patch-37

Update backup-azure-vms-troubleshoot.md

Commit:
Updated per team feedback

Lisaco88 committed:
Merge pull request #27175 from novino/patch-6

Removed wad section

davidbritch committed:
Update Xamarin.Forms authentication doc

Updated how to create the custom URL scheme on each platform.

cynthn committed:
Update convert-disk-storage.md

spelluru committed:
added a note

spelluru committed:
spelling fix

Lisaco88 committed:
Merge pull request #27110 from HJToland3/master

image update

Commit:
custom-search: add clarity and form

Lisaco88 committed:
Merge pull request #27313 from dominicbetts/suite-image-update

Update architecture diagram

PRMerger20 committed:
Merge pull request #27326 from davidbritch/patch-1

Update Xamarin.Forms authentication doc

PRMerger12 committed:
Merge pull request #27327 from cynthn/master

Update convert-disk-storage.md

ShawnJackson committed:
Merge pull request #26567 from ktoliver/storage-sync-files-change-detection

edit pass: includes/storage-sync-files-change-detection.md

PRMerger20 committed:
Merge pull request #27325 from brapel/custom-search-minor-updates

custom-search: add clarity and form

PRMerger18 committed:
Merge pull request #27328 from spelluru/adcrelationship1109

added a note

Lisaco88 committed:
Merge pull request #27319 from PatrickBue/master

Changed example images to fit on page

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #27200 from jeffgilb/jeff118

Updated Azure storage services comparison

v-alje committed:
Merge pull request #27331 from MicrosoftDocs/FromPublicRepo

From public repo

rmca14 committed:
Merge pull request #26569 from JPallavi/patch-31

Update backup-azure-configure-reports.md
